packages/framework/src/Resources/styles/admin/libs/selectize/selectize-custom.less

```diff
@@ -19,24 +19,44 @@
 @selectize-color-dropdown-item-active: @color-message-info;
 @selectize-color-dropdown-item-active-text: @color-base;
 @selectize-color-highlight: @color-message-info;
+@selectize-padding-input: 6px;
+@selectize-right-padding-input: 25px;
 
 .selectize-control {
-    height: 32px;
-    line-height: 32px;
+    height: @input-height;
+    line-height: @input-height;
```

Comment:
should not be here `@input-line-height` ???

Reply:
No. We want to vertical centering.